tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+
+- set_fact: "es_major_version={{ es_version[0] }}.x"
```

Comment:
This is gonna break when version 10 comes out ;).

Maybe it would be nicer to instead do something like `{{ es_version.split('.')[0] }}`